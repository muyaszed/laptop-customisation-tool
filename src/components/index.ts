export { default as SpecComponent } from "./SpecComponent/specComponent";
export { default as Header } from "./Header/header";
export { default as Content } from "./Content/content";
export { default as Box } from "./Box/box";
export { default as OptionItem } from "./OptionItem/optionItem";

export type { IComponentOption } from "./OptionItem/optionItem";
export type { IBoxProps } from "./Box/box";
