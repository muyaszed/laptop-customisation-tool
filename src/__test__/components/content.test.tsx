import renderer from "react-test-renderer";
import { Content } from "../../components";

describe("Home page", () => {
  test("render the page correctly by default", () => {
    const wrapper = renderer.create(<Content />);
    expect(wrapper).toMatchSnapshot();
  });
});
