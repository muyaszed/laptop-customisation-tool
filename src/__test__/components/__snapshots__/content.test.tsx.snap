// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page render the page correctly by default 1`] = `
<div
  className="content"
>
  <div
    className="spec-sheet rounded-grey-border"
    data-testid="spec-choices"
  >
    <h2>
      Customisation Choices
    </h2>
  </div>
  <div
    className="summary-window rounded-grey-border"
    data-testid="spec-summary"
  >
    <h2>
      Summary
    </h2>
    <div
      className="summary-window__chosen-specs"
    >
      <ul
        className="chosen-specs__spec-list list--no-style"
      />
    </div>
    <span
      className="summary-window__total"
    >
      Total: 
      Â£300
    </span>
    <button
      className="buy-button clickable"
      name="my-button"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      Buy now
    </button>
  </div>
</div>
`;
